examples/tests/sdk_tests/std/array.w

```diff
@@ -265,4 +265,21 @@ test "set()" {
   assertThrows(INDEX_OUT_OF_BOUNDS_ERROR, () => {
     mutArr.set(5, 11);
   });
+}
+
+//-----------------------------------------------------------------------------
+// insert()
+
+test "insert()" {
+  let arr = MutArray<num>[5, 10, 20];
+  assert(arr.at(2) == 20);
+
+  arr.insert(2, 15);
+
+  assert(arr.at(2) == 15);
+  assert(arr.at(3) == 20);
+
+  arr.insert(4, 25);
+
+  assert(arr.at(4) == 25);
```

Comment:
Can you please check the invalid cases too? (index below 0, or over arr.length)
and also, can you please check the array length after each insert? to make sure it increases by 1